src/optimum/nvidia/builder/local.py

```diff
@@ -89,6 +89,12 @@ def build_cli_command(
             "--logits_dtype": build_config.logits_dtype,
         }
 
+        if hasattr(model_config, "trt_model_class") and hasattr(
+            model_config, "trt_model_file"
+        ):
+            build_params["--model_cls_file"] = model_config.trt_model_file
+            build_params["--model_cls_name"] = model_config.trt_model_class
+
```

Comment:
In TensorRT-LLM, there is no class inheriting from `PretrainedModel`, [which is a requirement of `trtllm-build`](https://github.com/NVIDIA/TensorRT-LLM/blob/66ca3378c61efa3154ed34a48cfc362351405eef/tensorrt_llm/commands/build.py#L270). Hence, we define such a class in `whisper.py`